org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/text/correction/QuickAssistProcessor.java

```diff
@@ -230,7 +230,8 @@ public List<ProposalKindWrapper> getAssists(CodeActionParams params, IInvocation
 			getConvertToSwitchExpressionProposals(context, coveringNode, resultingCollections);
 
 			List<Integer> javaDocCommentProblems = Arrays.asList(IProblem.JavadocMissing);
-			if (!problemExists(locations, javaDocCommentProblems)) {
+			IProblemLocationCore found = findProblemLocation(locations, javaDocCommentProblems);

```

Comment:
Is there a reason to create `findProblemLocation` and not remove the other usage to `problemExists` ?

Reply:
Changed `problemExists` to call `findProblemLocation != null` instead.

Reply:
At this spot, you call `findProblemLocation` and store the problem as `found`, but where else are you using `found` except to check if it's `null`. Why not just use `problemExists` if you're not passing `found` anywhere else.